docs/content/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -247,68 +246,175 @@ You can see that certain tags include the version of Hadoop, e.g. (e.g. `-hadoop
 
-### Provide custom configuration
+* Scale the cluster up or down to `N` TaskManagers
 
-The configuration files (`flink-conf.yaml`, logging, hosts etc) are located in the `/opt/flink/conf` directory in the Flink image.
-To provide a custom location for the Flink configuration files, you can
+    ```sh
+    $ docker-compose scale taskmanager=<N>
+    ```
 
-* **either mount a volume** with the custom configuration files to this path `/opt/flink/conf` when you run the Flink image:
+* Access the JobManager container
 
     ```sh
-    $ docker run \
-        --mount type=bind,src=/host/path/to/custom/conf,target=/opt/flink/conf \
-        flink:{{< stable >}}{{< version >}}-scala{{< scala_version >}}{{< /stable >}}{{< unstable >}}latest{{< /unstable >}} <jobmanager|standalone-job|taskmanager>
+    $ docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
     ```
 
-* or add them to your **custom Flink image**, build and run it:
-
+* Kill the cluster
 
-    ```dockerfile
-    FROM flink
-    ADD /host/path/to/flink-conf.yaml /opt/flink/conf/flink-conf.yaml
-    ADD /host/path/to/log4j.properties /opt/flink/conf/log4j.properties
+    ```sh
+    $ docker-compose kill
     ```
 
-{{< hint info >}}
-The mounted volume must contain all necessary configuration files.
-The `flink-conf.yaml` file must have write permission so that the Docker entry point script can modify it in certain cases.
-{{< /hint >}}
+* Access Web UI
 
-### Using filesystem plugins
+  When the cluster is running, you can visit the web UI at [http://localhost:8081](http://localhost:8081).
 
-As described in the [plugins]({{< ref "docs/deployment/filesystems/plugins" >}}) documentation page: In order to use plugins they must be
-copied to the correct location in the Flink installation in the Docker container for them to work.
+### Application Mode
 
-If you want to enable plugins provided with Flink (in the `opt/` directory of the Flink distribution), you can pass the environment variable `ENABLE_BUILT_IN_PLUGINS` when you run the Flink image.
-The `ENABLE_BUILT_IN_PLUGINS` should contain a list of plugin jar file names separated by `;`. A valid plugin name is for example `flink-s3-fs-hadoop-{{< version >}}.jar`
+In application mode you start a Flink cluster that is dedicated to run only the Flink Jobs which have been bundled with the images.
+Hence, you need to build a dedicated Flink Image per application.
+Please check [here](#application-mode-on-docker) for the details.
+See also [how to specify the JobManager arguments](#jobmanager-additional-command-line-arguments) in the `command` for the `jobmanager` service.
 
-```sh
-    $ docker run \
-        --env ENABLE_BUILT_IN_PLUGINS=flink-plugin1.jar;flink-plugin2.jar \
-        flink:{{< stable >}}{{< version >}}-scala{{< scala_version >}}{{< /stable >}}{{< unstable >}}latest{{< /unstable >}} <jobmanager|standalone-job|taskmanager>
+<a id="app-cluster-yml">`docker-compose.yml`</a> for *Application Mode*.
```

Comment:
This results in a clickable link, but you can't download or do anything with it.

Reply:
I think was that is only an anchor.